Doc/library/multiprocessing.rst

```diff
@@ -598,6 +598,10 @@ The :mod:`multiprocessing` package mostly replicates the API of the
          acquired a lock or semaphore etc. then terminating it is liable to
          cause other processes to deadlock.
 
+   .. method:: kill()
```

Comment:
Since this is a new method, you need a `versionadded` marker a well (see above the `sentinel` attribute for an example).